core/gather/gatherers/link-elements.js

```diff
@@ -130,8 +130,20 @@ class LinkElements extends FRGatherer {
    * @return {Promise<LH.Artifacts['LinkElements']>}
    */
   static async getLinkElementsInHeaders(context, devtoolsLog) {
-    const mainDocument =
-      await MainResource.request({devtoolsLog, URL: context.baseArtifacts.URL}, context);
+    // We can _sometimes_ get the main resource in timespan mode.
```

Comment:
nit: I don't think this comment is true. We won't ever get the main document in timespan because `mainDocumentUrl` is always `undefined`